.circleci/config.yml

```diff
@@ -66,13 +66,17 @@ jobs:
   # Linux
   py36_linux:
     docker:
-      - image: circleci/python:3.6
+#      - image: circleci/python:3.6
+      # Temporary workaround around 3.6 image
+      - image: circleci/python@sha256:19a446501d3e0ca273ed3161eeca3780dadbc9e3f3a788c8fd2bc92f5ed9bed0
```

Comment:
This seems off, I noticed it again with your other diff.
it looks like you cherry picked the changes to your branch instead of rebasing it.

Reply:
Hm, I actually used rebase rather than cherry-picking. The `no_output_timeout` change is one you requested, and lines 67-71 are identical with master. Not sure what to do